packages/app-content-pages/src/screens/About/components/Contact.js

```diff
@@ -0,0 +1,114 @@
+
+const StyledButton = styled(Button)`
+  text-align: center;
+  color: white;
+  padding: 10px;
+  background: ${props => props.theme.global.colors['neutral-1']};
+  width: 400px;
+`
+
+export default function Contact({ widgetLoaded = false }) {
+  const { t } = useTranslation()
+
+  const handleClick = e => {
+    if (widgetLoaded) {
+      window.FreshworksWidget('open')
+    }
+  }
+
+  return (
+    <Box pad={{ horizontal: 'medium' }}>
+      <Heading
+        level={3}
+        size='1.5rem'
+        alignSelf='center'
+        margin='0'
+        weight='normal'
+      >
+        {t('AboutPage.contact.subheading')}
+      </Heading>
+      <Box
+        direction='row'
+        gap='xlarge'
+        width='100%'
+        justify='center'
+        pad={{ vertical: 'medium' }}
+      >
+        <Box width='30%' align='center'>
+          <Box
+            height='2.5rem'
+            width='2.5rem'
+            align='center'
+            justify='center'
+            background='neutral-1'
+            round='50%'
+          >
+            <Help size='1.5rem' color='white' />
+          </Box>
+          <Text textAlign='center' margin={{ top: 'xsmall' }}>
+            {t('AboutPage.contact.categories.one')}
+          </Text>
+        </Box>
+        <Box width='30%' align='center' pad={{ horizontal: 'small' }}>
+          <Box
+            height='2.5rem'
+            width='2.5rem'
+            align='center'
+            justify='center'
+            background='neutral-1'
+            round='50%'
+          >
+            <Group size='1.5rem' color='white' />
+          </Box>
+          <Text textAlign='center' margin={{ top: 'xsmall' }}>
+            {t('AboutPage.contact.categories.two')}
+          </Text>
+        </Box>
+        <Box width='30%' align='center'>
+          <Box
+            height='2.5rem'
+            width='2.5rem'
+            align='center'
+            justify='center'
+            background='neutral-1'
+            round='50%'
+          >
+            <Edit size='1.5rem' color='white' />
+          </Box>
+          <Text textAlign='center' margin={{ top: 'xsmall' }}>
+            {t('AboutPage.contact.categories.three')}
+          </Text>
+        </Box>
+      </Box>
+      <Paragraph margin={{ top: '0' }}>
+        <Trans
+          i18nKey='AboutPage.contact.paragraphs.one'
+          t={t}
+          components={[
+            <Anchor
+              key='publications-page'
+              href='/publications' // after switch to app-root, this will need to be /about/publications
+            />,
+            <Anchor
+              key='freshdesk-page'
+              href='https://zooniverse.freshdesk.com/support/solutions'
+            />
+          ]}
+        />
+      </Paragraph>
+      <Paragraph>{t('AboutPage.contact.paragraphs.two')}</Paragraph>
+      <Box align='center' pad={{ top: 'small', bottom: 'xlarge' }}>
+        <StyledButton primary textAlign='center' onClick={handleClick}>
```

Comment:
The "Contact Us" button is missing rounded corners like the Figma file.

Reply:
Fixed 👍